models/pancakeswap/pancakeswap_trades.sql

```diff
@@ -8,10 +8,10 @@
 }}
 
 {% set pancake_models = [
-    ref('pancakeswap_v2_bnb_trades')
-,   ref('pancakeswap_v2_ethereum_trades')
-,   ref('pancakeswap_v2_bnb_mmpool')
-,   ref('pancakeswap_v2_ethereum_mmpool')
+    ref('pancakeswap_v2_bnb_amm_trades')
+,   ref('pancakeswap_v2_ethereum_amm_trades')
+,   ref('pancakeswap_v2_bnb_mmpool_trades')
+,   ref('pancakeswap_v2_ethereum_mmpool_trades')
```

Comment:
Hi @chef-seaweed, you can also make a roll-up spell for each chain.

Reply:
Here is an example to follow:
https://github.com/duneanalytics/spellbook/blob/main/models/uniswap/ethereum/uniswap_ethereum_trades.sql